at-tanvo/D3-Exercise2.rb

```diff
@@ -0,0 +1,11 @@
+puts "Nhap phan tu mang :"
+a = gets.to_i
+arr = []
+for i in 1..a
+  b = gets.to_i
+  arr.push(b)  
+end
+puts "#{arr}"
+arr.map!{|i| i.even? ? 'even' : 'odd'}
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><table>
  <tbody>
    <tr>
      <td><span data-href="https://github.com/atruby-team/internship-18-summer/blob/5c4b6527b3afcb714c4f0ab1934ff6c19402ac0b/at-tanvo/D3-Exercise2.rb#L9"></span>Layout/SpaceInsideBlockBraces: Space between { and</td>
      <td>missing.</td>
    </tr>
  </tbody>
</table>
</td>
    </tr>
  </tbody>
</table>